src/site/xdoc/server/config-cassandra.xml

```diff
@@ -136,6 +136,11 @@
         <dd>Optional. Defaults to 50.<br/> Controls the number of messages to be expunged in parallel.</dd>
         <dt><strong>mailbox.blob.part.size</strong></dt>
         <dd>Optional. Defaults to 102400 (100KB).<br/> Controls the size of blob parts used to store messages.</dd>
+
+        <dt><strong>cassandra.consistency_level.regular</strong></dt>
+        <dd>Optional. Defaults to QUORUM.<br/> <a href="https://docs.datastax.com/en/cassandra-oss/3.x/cassandra/dml/dmlConfigConsistency.html">See possible values</a>.</dd>
```

Comment:
What is this value applied to? We miss a `Allows specifying the driver default consistency level.`